spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Cell.scala

```diff
@@ -180,3 +182,37 @@ abstract class Cell[T : ClassTag](
     }
   }
 }
+
+object CellSerializer extends ModuleSerializable {
+
+  override def loadModule[T: ClassTag](model : BigDLModule)
+                                      (implicit ev: TensorNumeric[T]) : ModuleData[T] = {
+    val moduleData = super.loadModule(model)
+    val cellModule = moduleData.module.asInstanceOf[Cell[T]]
+
+    val attrMap = model.getAttrMap
+    cellModule.cell = DataConverter.getAttributeValue(attrMap.get("cell")).
+      asInstanceOf[AbstractModule[Activity, Activity, T]]
+
+    createBigDLModule(model, cellModule)
+  }
+
+  override def serializeModule[T: ClassTag](module : ModuleData[T])
+                                           (implicit ev: TensorNumeric[T]) : BigDLModule = {
+    val bigDLModule = super.serializeModule(module)
+    val cellModule = module.module.asInstanceOf[Cell[T]]
+    val cellModuleBuilder = BigDLModule.newBuilder(bigDLModule)
+
+    val cellSerializerFlagBuilder = AttrValue.newBuilder
+    DataConverter.setAttributeValue(cellSerializerFlagBuilder, true,
+      scala.reflect.runtime.universe.typeOf[Boolean])
+    cellModuleBuilder.putAttr("is_cell_module", cellSerializerFlagBuilder.build)
```

Comment:
what's the is_cell_module used for? Looks like it is not used in the load method